File: backend-model/src/main/java/outcobra/server/model/dto/MarkReportDto.java
package outcobra.server.model.dto;

import outcobra.server.model.MarkReport;
import outcobra.server.model.interfaces.MappableDto;
import outcobra.server.model.interfaces.Mapper;

public class MarkReportDto implements MappableDto<MarkReportDto, MarkReport> {
    @Override
    public Mapper<MarkReportDto, MarkReport> getMapper() {
        throw new UnsupportedOperationException("not implemented");
    }

    @Override
    public MarkReport toEntity() {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: backend/src/main/kotlin/outcobra/server/model/dto/MarkReportDto.kt
package outcobra.server.model.dto

import noutcobra.server.model.mapper.MappableDto
import outcobra.server.model.MarkReport
import outcobra.server.model.mapper.Mapper

/**
 * Created by Florian on 04.11.2016.
 */
class MarkReportDto : MappableDto<MarkReportDto, MarkReport> {
    override fun getMapper(): Mapper<MarkReportDto, MarkReport> {
        throw UnsupportedOperationException("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun toEntity(): MarkReport {
        throw UnsupportedOperationException("not implemented") //To change body of created functions use File | Settings | File Templates.
    }
//TODO Implement
}